tests/e2e/feastoperator_test.go

```diff
@@ -23,8 +23,7 @@ func feastOperatorTestSuite(t *testing.T) { //nolint:unused
 	t.Run("Validate operands have OwnerReferences", componentCtx.ValidateOperandsOwnerReferences)
 	t.Run("Validate update operand resources", componentCtx.ValidateUpdateDeploymentsResources)
 	t.Run("Validate component disabled", componentCtx.ValidateComponentDisabled)
-	// TODO: Uncomment the following tests after yaml is in ODH repo
-	// t.Run("Validate component releases", componentCtx.ValidateComponentReleases)
+	t.Run("Validate component releases", componentCtx.ValidateComponentReleases)
```

Comment:
can this check be moved before  `Validate component disabled` ? so the component is not left running once the test ends

Reply:
Good catch thanks, fixed in latest commit